modules/rosapps/applications/vcdcontroltool/lang/uk-UA.rc

```diff
@@ -0,0 +1,62 @@
+﻿LANGUAGE LANG_UKRAINIAN, SUBLANG_NEUTRAL
+
+IDD_MAINWINDOW DIALOGEX 0, 0, 320, 200
+STYLE DS_SETFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
+EXSTYLE WS_EX_APPWINDOW
+CAPTION "Панель керування віртуальним CDRom"
```

Comment:
"CDRom" here, but "CD-ROM" in line 44 ?

Reply:
In original text (`en-US.rc`) written "CDRom". Do translators have to correct these errors?

Reply:
Fixed.